tests/integration/testdata/migration/secrets/01G6K8CZ08T78WXTJYHR0NTYBS-name

```diff
@@ -0,0 +1 @@
+example-bundle
```

Comment:
If the name isn't defined as sensitive, I'm a bit confused on why the secret plugin still created a secret for this parameter

Reply:
There is both a parameter "name" and an output "name". The parameter is not sensitive. The output is though

https://github.com/getporter/examples/blob/571103890795923800762b132ac7b22afb28ff28/sensitive-data/porter.yaml#L18-L21

So we have one sensitive parameter, "password" and one sensitive output "name" which is why we got 2 secrets in the filesystem plugin.

Maybe the example bundle could be improved for clarity if we changed the output to be admin-password (instead of name) and have it persist the password parameter value? I think part of the problem may be that the parameter isn't sensitive but the output is and they both have the same name.